### PR TITLE
FLUT-6833 - [Bug] Resolved incorrect code snippet of export to pdf user guide documentation in flutter charts

### PR DESCRIPTION
**Pull request by SriramKiranSenthilkumar**
**Description:**

- Resolved the incorrect code snippet provided in the export to pdf user guide documentation section in the cartesian and circular charts.

### CODE REVIEW

**Review by LavanyaGowtham2021 — APPROVED** [review]
Changes are fine.

**Comment by LavanyaGowtham2021** [comment]
@SriramKiranSenthilkumar Add gif instead of image for exporting ug.

**Comment by SriramKiranSenthilkumar** [comment]
> @SriramKiranSenthilkumar Add gif instead of image for exporting ug.Hi @LavanyaGowtham2021,

I have added the gif images for the pdf exporting UG sections. Please check.

-Sriram Kiran

File: Flutter/funnel-chart/export-funnel-chart.md [hunk 1]
```diff
@@ -18,57 +18,246 @@ To export the funnel chart as a PNG image, we can get the image by calling [`toI
...
+    class ChartSampleData {
+      ChartSampleData({this.x, this.y});
+        final String? x;
+        final num? y;
+    }
+
+{% endhighlight %}
 {% endtabs %}
 
+![image_export](images/export-funnel-chart/image_view.gif)
```
Comment (VijayakumarMariappan) [inline Flutter/funnel-chart/export-funnel-chart.md:123]: image

Form a git like this

**Review by VijayakumarMariappan — CHANGES_REQUESTED** [review]

**Review by LavanyaGowtham2021 — APPROVED** [review]
Changes are fine.

**Review by VijayakumarMariappan — APPROVED** [review]
Changes are fine

**Pull request merged**